os_stub/cryptlib_openssl/pk/x509.c

```diff
@@ -2156,3 +2160,338 @@ bool libspdm_x509_get_cert_from_cert_chain(const uint8_t *cert_chain,
 
     return false;
 }
+
+bool libspdm_set_subject_name(X509_NAME *x509_name, char *subject_name)
+{
+    int ret;
+    uint8_t index;
+    char *char_start;
+    char *char_end;
+    char temp[MAX_SBUJECT_LEN];
+
+    ret = 0;
+    char_start = NULL;
+    char_end = NULL;
+
+    char *standard_subject[] = {"C=","O=","L=","CN=","OU=","ST="};
+    char *subject_set[] = {"C","O","L","CN","OU","ST"};
```

Comment:
Is this a full list?
Please describe where it is documented.

Reply:
I have added the full list and reference spec.